main.js

```diff
@@ -1,46 +1,27 @@
-const {app, BrowserWindow} = require('electron')
-const path = require('path')
-const url = require('url')
-const pyshell =  require('python-shell');
+/* eslint-disable-next-line */
+const { app, BrowserWindow } = require('electron');
 
+// Connecting with our web app to use the same API
+require('./index');
 
+// Initializing the electron app
 
-function createWindow () {
-    window = new BrowserWindow({
-      width: 800, 
-      height: 600
-    })
-    //window.loadURL('https://youtube.com')
-    //console.log(`Loading: file://${__dirname}/public/index.html`);
-    
-    window.loadURL(`file://${__dirname}/index.html`)
-    window.setAlwaysOnTop(true)
+function createWindow() {
+  const window = new BrowserWindow({
+    width: 800,
```

Comment:
what is the unit here?

Reply:
this is the default size of the window, I just kept it as the default given by the Electronjs docs for now